packages/nexus/src/token.ts

```diff
@@ -0,0 +1,78 @@
+const OPERATION_TOKEN_TYPE_WORKFLOW_RUN = 1;
+
+// OperationTokenType is used to identify the type of operation token.
+// Currently, we only have one type of operation token: WorkflowRun.
+type OperationTokenType = typeof OPERATION_TOKEN_TYPE_WORKFLOW_RUN;
+
+interface WorkflowRunOperationToken {
```

Comment:
Will you still be happy with this name when we have other operations such as `WorkflowSignalWithStartOperation` and `WorkflowUpdateWithStartOperation` that start workflows? Or is it better to give it a slightly more general name that indicates that it represents a workflow but is agnostic regarding how that workflow was started?